react/components/BuyButton/index.js

```diff
@@ -51,25 +55,28 @@ const skuItemToMinicartItem = item => {
  * Adds a list of sku items to the cart.
  */
 export const BuyButton = ({
-  isOneClickBuy = false,
-  shouldOpenMinicart = false,
-  available = true,
   intl,
+  large,
   addToCart,
-  setMinicartOpen,
   skuItems,
   onAddStart,
   onAddFinish,
+  setMinicartOpen,
+  available = true,
   orderFormContext,
-  children,
-  large,
-  disabled,
+  isOneClickBuy = false,
+  children: childrenProp,
+  disabled: disabledProp,
```

Comment:
Because I wanna use the `disabled` name later [here](https://github.com/vtex-apps/store-components/pull/625/files#diff-055ae6de82e52edc20b8c409a9b0f7f4R203)